src/dmd/frontend.d

```diff
@@ -21,6 +21,11 @@ import std.traits : isNarrowString;
 version (Windows) private enum sep = ";", exe = ".exe";
 version (Posix) private enum sep = ":", exe = "";
 
+shared static this()
+{
+    initDMD();
+}
```

Comment:
I'm not sure I like this. The original reason why it's necessary to call `initDMD` is that I said no to using a module constructor. What I said back then was that there might be a use case/possible to use multiple instances of the compiler in the future. I'm not sure if that's worth strive for.

An alternative would be to wrap everything in a class that needs to be instantiated, `Frontend` or `Compiler`.

Reply:
Yes, we should strive for making the compiler as reusable as a simple object with constructor and destructor, so +1 for having a `Compiler` class/struct.

Reply:
Not sure whether that makes sense as long as the compiler still uses globals - it would create a false abstraction.
On the other hand, we need to start somewhere and we could already store state like the import paths...

Reply:
> Not sure whether that makes sense as long as the compiler still uses globals - it would create a false abstraction.

True. But changing the API (if needed in the future) is usually quite difficult. Another idea is to use a singleton. The user basically gets access to an instance but can not create new instances. Later we can lift that restriction and I don't think that would break the API.

Reply:
I am quite aware that we're pretty far from having a reusable `Compiler` RAII object. I'm just saying that we should *strive* to change the architecture.